src/components/Accordion/docs/Accordion.stories.mdx

```diff
@@ -116,7 +134,8 @@ A scrollbar has to be added in the content after exceeding `37rem`.
 
 ## States
 
-The accordion component is by default in the collapsed state with all content panels closed.
+By default, only the first panel of the accordion component is opened.
+Only one panel can be opened at a time.
```

Comment:
The previous guidelines were not applicable using the component: it forces a single panel to be opened at all times.